src/pages/help/index.jsx

```diff
@@ -25,13 +25,14 @@ class HelpPage extends React.Component {
   }
 
   search(e) {
-    this.setState({
-      searchQuery: e.target.value,
-    })
-    setTimeout(() => {
-      index.search(this.state.searchQuery, (err, content) => {
-        this.setState({
-          hits: content.hits,
+    this.setState({ searchQuery: e.target.value }, () => {
+      setTimeout(() => {
```

Comment:
Doesn't look like `setTimeout` would be needed any more.
Probably was required before to ensure `searchQuery` is updated.